secret_number.js

```diff
@@ -1,4 +1,40 @@
+//var secretNumber = function(){
 'use strict';
 module.exports = function() {
-  
+
+
+var sameRandomNumber = 0;
+var genRandomNumber = Math.random();
```

Comment:
This could be simplified. I don't think you need both sameRandomNumber and genRandomNumber.